backend/routes/_auth.js

```diff
@@ -45,7 +45,7 @@ async function authSellerAndShop(req, res, next) {
   }
 
   if (req.seller.superuser) {
-    Shop.findOne({ where: { authToken } }).then((shop) => {
+    return Shop.findOne({ where: { authToken } }).then((shop) => {
```

Comment:
super nit: maybe we should use the await syntax in this route like the rest of this file. I personally alway find these .then callbacks hard to read/reason about.